src/main/java/org/ays/auth/service/AysRoleReadService.java

```diff
@@ -19,6 +21,13 @@
  */
 public interface AysRoleReadService {
 
+    /**
+     * Retrieves all roles.
+     *
+     * @return A set of {@link AysRole} objects representing all roles in the system.
+     */
+    Set<AysRole> findAll();
```

Comment:
Set dönmek yerine `List` dönebiliriz. MapStruct kütüphanesinde fark ettiğimiz bir problem vardı. Bu sebeple spesifik ihtiyacımız olmadığı sürece `Set` yerine `List` kullanmamız daha sağlıklı olabilir.

```suggestion
     * @return A list of {@link AysRole} objects representing all roles in the system.
     */
    List<AysRole> findAll();
```

Reply:
değişikliği yaptım